Expensify/App
# Enlarge image modal and update some styling

<Assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

<Explanation of the change or anything fishy that is going on>

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/146568
Fixes https://github.com/Expensify/Expensify/issues/146475

### Tests
<Tests you performed on all platforms that validates your changed worked>

1. Upload an image
2. Click on the image and notice that the image modal is much larger (takes up about 90% of the screen)

### Screenshots


![Screen Shot 2020-12-09 at 10 48 42 AM](https://user-images.githubusercontent.com/30609178/101673575-351c5580-3a0c-11eb-8052-5f66ff881d2b.png)

![Screen Shot 2020-12-09 at 10 50 55 AM](https://user-images.githubusercontent.com/30609178/101673749-6d239880-3a0c-11eb-9602-0b2748cb3d15.png)

Mobile header updates:
![Screen Shot 2020-12-11 at 11 55 36 AM](https://user-images.githubusercontent.com/30609178/101949144-fe7b4200-3ba7-11eb-9d30-deb12747a918.png)
![Screen Shot 2020-12-11 at 11 43 47 AM](https://user-images.githubusercontent.com/30609178/101949155-00450580-3ba8-11eb-87ac-7e121488e521.png)
